packages/changed-elements-react/src/NamedVersionSelector/Sticky.tsx

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import clsx from "clsx";
+import {
+  useLayoutEffect, useRef, useState, type ReactElement, type ReactNode, type RefObject
+} from "react";
+
+import "./Sticky.css";
+
+interface StickyProps {
+  className?: string | undefined;
+  children?: ReactNode | undefined;
+}
+
+export function Sticky(props: StickyProps): ReactElement {
+  const ref = useRef<HTMLDivElement>(null);
+  const stuck = useSticky(ref);
+
+  return (
+    <div ref={ref} className={clsx("_cer_v1_sticky", props.className)} data-stuck={stuck}>
+      {props.children}
+    </div>
+  );
+}
+
+function useSticky(ref: RefObject<HTMLElement>): boolean {
```

Comment:
Could you add documentation for this? There are also a lot of other functions in the PR that are lacking any docs.